packages/assets-controllers/src/Standards/NftStandards/ERC721/ERC721Standard.ts

```diff
@@ -187,20 +139,16 @@ export class ERC721Standard {
     address: string,
     interfaceId: string,
   ): Promise<boolean> => {
-    const contract = this.web3.eth.contract(abiERC721).at(address);
-    return new Promise<boolean>((resolve, reject) => {
-      contract.supportsInterface(
-        interfaceId,
-        (error: Error, result: boolean) => {
-          /* istanbul ignore if */
-          if (error) {
-            reject(error);
-            return;
-          }
-          resolve(result);
-        },
-      );
-    });
+    const contract = new Contract(address, abiERC721, this.provider);
+    try {
+      return await contract.supportsInterface(interfaceId);
+    } catch (err: any) {
+      // Mirror previous implementation
+      if (err.message.includes('call revert exception')) {
```

Comment:
I'm guessing this was what web3 did internally. Do you know offhand which line that was on?

Reply:
I do not, but this mirrors that the tests are expecting. We can remove it but that would be a breaking change.